tele_giphy/tele_giphy/urls.py

```diff
@@ -18,6 +18,7 @@
 
 urlpatterns = [
     url(r'^', include('game.urls')),
+    url(r'^game/', include('game.urls')),
```

Comment:
Two urls pointing to the same place? What's the plan here? I dunno if it's good or bad, never seen it before though